app/assets/stylesheets/components/vacancy_review_heading.scss

```diff
@@ -0,0 +1,14 @@
+@import 'base';
+
+.vacancy-review-heading-component {
```

Comment:
this naming should really follow the 'root' component i.e `review-component__heading` and then you can use ampersand notation `&__heading`